src/components/pumps/pump-tooltip.tsx

```diff
@@ -43,11 +43,11 @@ export const PumpTooltip: React.FC<PumpTooltipProps> = ({ pump }) => {
 					<div className="">{pump.lastCheck}</div>
 				</div>
 			</div>
-			<div className="text-sm pt-2">
+			{/* <div className="text-sm pt-2 border-t-2 ">
 				<a className="underline" href="">
 					{i18n.pumps.update}
 				</a>
-			</div>
+			</div> */}
```

Comment:
let's add a comment why we want to keep this code comment, otherwise after relooking at this in some time we might be unaware of why we keep it and delete it